arc/cli/utils/state.py

```
@@ -70,25 +70,66 @@ def __init__(self, state_dir: Optional[Path] = None):
         
         # Create lock file if it doesn't exist
         self.lock_file.touch(exist_ok=True)
         
+        # Check for and clean up stale locks
+        self._cleanup_stale_lock()
+        
         with open(self.lock_file, 'w') as lock_fd:
+            # Write current process PID for stale lock detection
+            lock_fd.write(f"{os.getpid()}\n{int(time.time())}\n")
+            lock_fd.flush()
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)